source/SIL.AppBuilder.Portal.Frontend/src/data/containers/with-current-organization/hoc.ts

```diff
@@ -0,0 +1,61 @@
+import { withData, WithDataProps } from 'react-orbitjs';
+import { connect } from 'react-redux';
+import {
+  compose, withProps, branch, renderComponent
+} from 'recompose';
+
+import { State } from '@store/reducers';
```

Comment:
something else to add to linting... we should change that to IState to not get confuse

Reply:
aye, I'll make a PBI

Reply:
https://developertown.visualstudio.com/SIL%20International%20Scriptoria/_workitems/edit/30645